### PR TITLE
[IMP] sale_order_mass_action: Add missing target states

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

File: sale_order_mass_action/wizards/sale_order_mass_action_wizard.py [hunk 1]
```diff
@@ -4,22 +4,50 @@
...
+        domain = expression.AND([domain, self._get_sale_order_state_domain()])
+        return domain
+
+    def _launch_action(self, sale_orders):
+        self.ensure_one()
+        action = "action_" + self.action
+        if hasattr(sale_orders, action):
+            getattr(sale_orders, action)()
+            return True
+        return False
```
Comment (legalsylvain) [inline sale_order_mass_action/wizards/sale_order_mass_action_wizard.py:50]: minor question (not blocking) : if the action is not found in the hasattr, I expect the function raise an error. don't you think ?

**Review by legalsylvain — APPROVED** [review]
LGTM. Thanks !

File: sale_order_mass_action/wizards/sale_order_mass_action_wizard.py (hunk 1 above)
Comment (rousseldenis) [reply sale_order_mass_action/wizards/sale_order_mass_action_wizard.py:50]: Maybe. In this case, the user won't have the result notification with the updated sale orders.

But we could improve it and display an error notify in that case

Comment (rousseldenis) [reply sale_order_mass_action/wizards/sale_order_mass_action_wizard.py:50]: @legalsylvain What do you think ?

**Comment by Garibaldy** [comment]
why this pull request is not merge yet?
 i'm considering to migrate it to v15  ASAP

File: sale_order_mass_action/wizards/sale_order_mass_action_wizard.py (hunk 1 above)
Comment (legalsylvain) [reply sale_order_mass_action/wizards/sale_order_mass_action_wizard.py:50]: But in which case the action doesn't exist ? Sorry but i dont understand that test.

**Pull request closed without merging**